wordpress-mobile/WordPress-Android
# Remove unused threaded comment class

This PR basically reverts the #15587 since those classes are unused for now and makes sense to clean it up.

You can use the diff from that PR as a guideline to track modifications back.

## To test
- Check unit tests are green
- Smoke test the threaded comments functionality (pagination, like, PTR, reply to comment and post, Follow conversation by email/push, anything else you can think of....) checking no regression was introduced.
- Test deep linking to open a comment or like a comment. You should be landed in the threaded comments with the eventual action (for example like) applied.

## Regression Notes
1. Potential unintended areas of impact
Should be none

2. What I did to test those areas of impact (or what existing automated tests I relied on)
No unintended area of impact should be there; anyways we are relying on already available unit testing and manual testing.

3. What automated tests I added (or what prevented me from doing so)
This is removing unused class so no tests addition needed.

PR submission checklist:
